docs/faq/DAGMan/bagman-exit-at-percent-done.rst

```diff
@@ -0,0 +1,149 @@
+Example: DAGMan Exit Upon Percent Completed
+===========================================
+
+.. sidebar:: Definition: Bag of Nodes
+
+    A bag of nodes is a DAG workflow comprising of nodes bearing
+    no :dag-cmd:`PARENT/CHILD` relationships. Thus, being a loose bag of nodes
+    without any inter-dependencies.
+
+Sometimes it is desired to have a DAG set up as a bag of nodes that stops
+executing after a certain percentage of successfully executed nodes. The following
```

Comment:
again "completes, or finishes, instead of "exits".  I think "exit" might imply "some kind of error" to some users.